### ISSUE TITLE
Extension request: definition lists

### ISSUE THREAD

**Issue by trombonehero** [russross/blackfriday#144]
[kramdown](http://kramdown.gettalong.org/syntax.html#definition-lists) and [Maruku](https://github.com/bhollis/maruku/blob/master/docs/maruku.md) both support a rather useful definition list syntax:

```markdown
Alice
: the first participant in a protocol
Bob
: the second participant in a protocol
Eve
: a passive eavesdropper
```

Is there any chance that such an extension could be added to blackfriday (even if disabled by default)?

**Comment by miekg** [comment]
I've implemented this in my (personal) fork: https://github.com/miekg/mmark/blob/master/block.go#L216

**Comment by rtfb** [comment]
Implemented in #171.

**Comment by shurcooL** [comment]
This appears to be a duplicate of #62, which is therefore also resolved.
